packages/next/build/webpack-config.js

```diff
@@ -291,6 +291,7 @@ export default async function getBaseWebpackConfig (dir, {dev = false, isServer
       dev && new UnlinkFilePlugin(),
       dev && new CaseSensitivePathPlugin(), // Since on macOS the filesystem is case-insensitive this will make sure your path are case-sensitive
       !dev && new webpack.HashedModuleIdsPlugin(),
+      process.env.NODE_ENV === 'production' && new webpack.IgnorePlugin(/^react-is$/),
```

Comment:
It actually has to be more like: https://github.com/zeit/next.js/blob/master/build/webpack.js#L252-L261

Because a dependency could use `react-is` without checks

Reply:
```
!dev && new webpack_1.default.IgnorePlugin({
                checkResource: (resource) => {
                    return /react-is/.test(resource)
                },
                checkContext: (context) => {
                    return /next-server[\\/]dist[\\/]/.test(context) || /next[\\/]dist[\\/]/.test(context)
                }
            }),
```

This works